controllers/community_modules.py

```diff
@@ -5,26 +5,40 @@
 # try something like
 def index(): return dict(message="hello from communities_modules.py")
 
-# Create a new community module from a given payload. Throws an error if no payload is given, or the community module already exists.
-def create():
-    payload = request.body.read()
-    if not payload:
-        return dict(msg="No payload given.")
-    payload = json.loads(payload)
-    if 'module_id' not in payload or 'community_id' not in payload:
-        return dict(msg="Payload missing required fields.")
-    if db((db.community_modules.community_id == payload['community_id']) & (db.community_modules.module_id == payload['module_id'])).count() > 0:
-        return dict(msg="Community Module already exists.")
-    # Check if enabled is in the payload, if not, add it with a default value of True
-    if 'enabled' not in payload:
-        payload['enabled'] = True
+def get_identity(identity_name):
```

Comment:
Need to valid the type at minimum , for example:
```suggestion
def get_identity(identity_name: str):
```

Reply:
Ok

This needs to happen on a bunch of controllers, so i figured when everything is merged, ill create a new branch for this to fix all these input types at once